### ISSUE TITLE
Do we need sourcemaps in production build?

### ISSUE THREAD

**Issue by Trost123** [yeoman/generator-webapp#599]
I just noticed that main.css (and main.js) have sourcemaps embedded in dest folder.
Do we need them on production?
If not, it can be fixed by modifying 2 lines in scripts and styles tasks:
```
    .pipe($.if(dev, $.sourcemaps.init()))
    .pipe($.if(dev, $.sourcemaps.write()))
```

**Comment by silvenon** [comment]
Great question, I don't think we need them in production. Feel free to open a PR 😉

**Comment by silvenon** [comment]
Fixed in [v4.0.0-0](https://github.com/yeoman/generator-webapp/releases/tag/v4.0.0-0)!
